README.markdown

```diff
@@ -240,6 +241,14 @@ Sets the timeout (in ms) protection for subsequent operations, including the `co
 
 [Back to TOC](#table-of-contents)
 
+set_timeouts
+----------
```

Comment:
Better align with the previous line.

Reply:
align it is.

Reply:
The "----" thing is still not aligned up with `set_timeouts` yet?

Reply:
you mean the title length should be eq with short-line count?